src/main/java/io/fabric8/elasticsearch/plugin/osclient/Fabric8OpenShiftClientAdapter.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.fabric8.elasticsearch.plugin.osclient;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import io.fabric8.kubernetes.client.ConfigBuilder;
+import io.fabric8.openshift.api.model.Project;
+import io.fabric8.openshift.client.DefaultOpenshiftClient;
+
+public class Fabric8OpenShiftClientAdapter implements OpenShiftClient {
```

Comment:
Originally added adapter when I was trying to make it work and pulled in the openshift-restclient-java.  This was before project retrieval was supported.  I'll remove it.